gc-worker/Makefile

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build pd-server, pd-ctl, pd-recover
+default: build
+
+GO             := GO111MODULE=on go
+BUILD_BIN_PATH := $(shell pwd)/bin
+PACKAGES       := go list ./...
+DIRECTORIES    := $(PACKAGES) | sed 's|github.com/tikv/migration/gc-worker/||'
+
+#### Build ####
+build: gc-worker
```

Comment:
Create a target for unit tests.
Please create Github actions for CI. Refer to `.github/workflows/ci-cdc.yml`.

Reply:
done, build and test actions are added.